UserController

```diff
@@ -0,0 +1,9 @@
+[HttpPost("delete/{id}")]
+public void Delete(uint id)
```

Comment:
Name is too vague and I would make the method async - rename it to DeleteUserAync and ensure the proper async methods are used